addons/hr_recruitment/views/hr_applicant_views.xml

```diff
@@ -82,14 +82,6 @@
                             <span class="o_stat_text">Employee</span>
                         </div>
                     </button>
-                    <button name="action_open_similar_candidates"
-                            class="oe_stat_button"
-                            icon="fa-pencil"
-                            type="object"
-                            context="{'active_test': False}"
-                            invisible="similar_candidates_count == 0">
-                        <field name="similar_candidates_count" widget="statinfo" string="Similar Candidates"/>
-                    </button>
```

Comment:
Why remove the button here?

Reply:
Thank you for the review(and all the other reviews from the past days)!
So the new spec indicates that you should only be able to view similar candidates from the candidate form. Also the `Other Applications` button will now display all the other applications by the same candidate AND all other applications by similar candidates (basically combining the old `Similar Candidates` and `Other Application` buttons into one).